packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -794,7 +794,7 @@ class _TimePickerDialogState extends State<_TimePickerDialog> {
 ///  * <https://material.google.com/components/pickers.html#pickers-time-pickers>
 Future<TimeOfDay> showTimePicker({
   BuildContext context,
```

Comment:
Does `@required` always mean _non null_ in flutter code base ? IIRC I saw some case with a `@required` but without `assert(xxx != null)`